### ISSUE TITLE
API key not accepted

### ISSUE THREAD

**Issue by BliXem1** [briis/weatherbit#54]
Hi,

I already have a account at weatherbit, but I reset my API four times now, the master API key. But when I want to add this in Home Assistant it won't accept the key. Any idea? Installed your 'app' via HACS.
Error is: API key not correct.

**Comment by briis** [comment]
It is hard for me to say what is going wrong - The error message says it all - the API key you use to try and get data is not accepted by WeatherBit, for some reason.

If you go here: https://www.weatherbit.io/api/swaggerui/weather-api-v2#/Current32Weather32Data you can test getting data with your key, and if that works, the Integration should also work.

**Comment by briis** [comment]
Did you solve the problem?

**Comment by briis** [comment]
I will close this now. Please reopen if you have more data to share.

**Comment by bachoo786** [comment]
Same issue here on HA it says the api key is invalid.

**Comment by bachoo786** [comment]
This is what I get in the link you posted above:

{
  "error": "API key not valid, or not yet activated."
}

**Comment by briis** [comment]
This is out of my hands. Something is wrong with your key or with the Weatherbit backend.
I just checked with my key, and that works fine, so something is not right with your key.

**Comment by kdw2060** [comment]
update: can remain closed. The try-out via swagger seems to have somehow 'revived' my api key, is being accepted in integration config now too. Leaving this comment here in case other users experience the same.

~~Hi, I'd like to reopen this issue.
Removed and reinstalled your integration > configuring again > same error about api key not being valid.
I have tested my api key via the swagger api link you provided and it works fine there without errors.~~
